tests/test_dbapi2.py

```diff
@@ -43,7 +42,7 @@
 try:
     from unittest.mock import patch
 except ImportError:
-    from mock import patch
+    from unittest.mock import patch
```

Comment:
We should just remove the try/except now - every version we support will now have `unittest.mock` in the stdlib, so we'll never get this `ImportError`